resources/asciidoctor/spec/chunker_spec.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+require 'chunker/extension'
+require 'fileutils'
+require 'tmpdir'
+
+RSpec.describe Chunker do
+  before(:each) do
+    Asciidoctor::Extensions.register Chunker
+  end
+
+  after(:each) do
+    Asciidoctor::Extensions.unregister_all
+  end
+
+  include_context 'convert without logs'
+  let(:backend) { :html5 }
+
+  context 'when outdir is configured' do
+    let(:outdir) { Dir.mktmpdir }
+    after(:example) { FileUtils.remove_entry outdir }
```

Comment:
This looks totally fine, but just to share something I've used in the past: the `around` hook can bring up/teardown temp files in one block to consolidate where that's handled to avoid any weirdness if lazy `let` variables don't get evaluated or something else in between the `let` and `after`. [Example](https://gist.github.com/sue445/d840d6f68771a14a48fc)

Reply:
I do that sort of thing in the integration tests because I want to be super careful about building things one time. But I try to go traditional in the unit tests because it is still quite fast and makes ruby folks feel a little more at home.